cv_origami/utils/file.py

```diff
@@ -43,3 +47,58 @@ def get_log_path(log_file):
         return log_file_path
     else:
         return None
+
+
+def extract_zip_to_dir(zip_path, extract_path):
```

Comment:
Add checks for all kinds of things. There should be a size limit. Also, it should be valid zip. Check that before extracting. These are the ones I can think of currently, add other checks too which come to your mind.

Reply:
This method assumes that all the path we are providing to it have been validated earlier, like here https://github.com/Cloud-CV/origami-daemon/pull/5/files/5519e460a5a96f8694a226a54e6ff5fa4ac6c779#diff-e09c21be2fdca051065494a283e98131R131

I will add checks for size too.

Reply:
@uttu357 done :)